phylanx/execution_tree/primitives/base_primitive.hpp

```diff
@@ -218,6 +222,8 @@ namespace phylanx { namespace execution_tree { namespace primitives
     protected:
         static std::vector<primitive_argument_type> noargs;
         std::vector<primitive_argument_type> operands_;
+        mutable std::int64_t count_;
+        mutable std::int64_t duration_;
```

Comment:
Those new variables are not initialized anywhere

Reply:
Resolved.